core/src/main/java/hudson/model/ChoiceParameterDefinition.java

```diff
@@ -51,6 +56,58 @@ private ChoiceParameterDefinition(String name, List<String> choices, String defa
         this.defaultValue = defaultValue;
     }
 
+    /**
+     *
+     * @param name
+     * @param description
+     *
+     * @since TODO
+     */
+    @DataBoundConstructor
+    @Restricted(NoExternalUse.class) // there are specific constructors with String and List arguments for 'choices'
+    public ChoiceParameterDefinition(String name, String description) {
```

Comment:
`description` should really be in a `@DataBoundSetter`, preferably pulled up to `ParameterDefinition`, but does not need to be done here I guess.